tests/test_cases/issue_608/issue_608.py

```diff
@@ -0,0 +1,43 @@
+"""Test case for BinaryValue behaviour documented in issue #608
```

Comment:
This test was also supposed to test other representations - 2's complement and signed magnitude. Is there a reason that this part has been removed? Here is the original test: https://github.com/cocotb/cocotb/pull/706/files#diff-762fb8676ef2834a87740564e711161d